deploy/crds/k8s_v1alpha1_redis_crd.yaml

```diff
@@ -81,6 +86,11 @@ spec:
               required:
               - image
               type: object
+            inits:
```

Comment:
I believe it is better to call it `initContainers` to avoid ambiguity.

Reply:
Done.